pkgs/tools/security/enpass/default.nix

```diff
@@ -0,0 +1,80 @@
+{stdenv, fetchurl, dpkg, patchelf, openssl, xorg
+, glib, mesa, libpulseaudio, zlib, dbus, fontconfig, freetype
+, gtk2, pango, atk, cairo, gdk_pixbuf, jasper, xkeyboardconfig
+, makeWrapper , makeDesktopItem, ...}:
+let
+  data = import ./data.nix;
+  uselibs = with xorg; [
+    openssl.out
+    mesa
+    fontconfig
+    freetype
+    libpulseaudio
+    zlib
+    dbus
+    libX11
+    libXi
+    libSM
+    libICE
+    libXext
+    libXrender
+    libXScrnSaver
+    glib.out
+    gtk2
+    pango
+    cairo
+    atk
+    gdk_pixbuf
+    jasper
+  ];
+    libPath = stdenv.lib.makeLibraryPath ([
+      stdenv.cc
+      stdenv.cc.cc
+    ] ++ uselibs);
+in stdenv.mkDerivation rec {
+
+    version = data.version;
+    name = "enpass-${version}";
+
+    desktopItem = makeDesktopItem {
+      name = "Enpass";
+      exec = "$out/bin/Enpass";
+      #icon = "Enpass";
+      comment = "password manager";
+      desktopName = "Enpass";
+      genericName = "Password manager";
+      categories = "Application;Development;";
+    };
+
+
+    src = fetchurl {
+      url = "http://repo.sinew.in/${data.path}";
+      sha256 = data.sha256;
+    };
+
+    buildInputs = [makeWrapper dpkg patchelf ];
+    phases = [ "unpackPhase" "installPhase" "fixupPhase" ];
+
+    unpackPhase = "dpkg -X $src .";
+    installPhase=''
+    mkdir $out
+    cp -r opt/Enpass/*  $out
+
+    # Make desktop item
+    mkdir -p "$out"/share/applications
+    cp "$desktopItem"/share/applications/* "$out"/share/applications/
+    mkdir -p "$out"/share/icons
+
+    patchelf  \
+      --set-interpreter $(cat $NIX_CC/nix-support/dynamic-linker) \
+      $out/bin/Enpass
+
+    wrapProgram $out/bin/Enpass \
+      --set LD_LIBRARY_PATH "${libPath}:$out/lib:$out/plugins/sqldrivers" \
+      --set QT_PLUGIN_PATH "$out/plugins" \
+      --set QT_QPA_PLATFORM_PLUGIN_PATH "$out/plugins/platforms" \
+      --set QT_XKB_CONFIG_ROOT "${xkeyboardconfig}/share/X11/xkb"
+    '';
+
+
```

Comment:
looks ok to me now, maybe add yourself as maintainer if you want to increase the chances of getting this merged